templates/base.html

```diff
@@ -90,7 +91,6 @@
               </a>
               <div class="dropdown-menu" aria-labelledby="navbarDropdownMenuLink">
                 <a class="dropdown-item" href="{{ config.base_url | safe }}/communities/map/">Map</a>
-                <a class="dropdown-item" href="{{ config.base_url | safe }}/communities/members/">Members</a>
```

Comment:
should the members.md file be deleted since it's not reachable anymore?

Reply:
I intentionally did not in this PR, but I don't mind if you would like me to remove it here.